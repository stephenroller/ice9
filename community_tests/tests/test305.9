# recursive stack limit test (int ops) - fires on 99

proc foo()
	var i : int
	
	i := 100 / 3 + 56 % 7 - 3;
	
	foo();
end

foo();
