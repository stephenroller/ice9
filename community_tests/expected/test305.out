Loading file: generated/test305.tm
Out of stack space 7
Number of instructions executed = 716
